docs/miscellaneous.rst

```diff
@@ -456,3 +456,21 @@ Language Grammar
 
 .. literalinclude:: grammar.txt
    :language: none
+
+**********************
+Error Recovery Grammar
```

Comment:
I think without context, this is not really understandable and could be better to leave this documentation for a later PR.